src/web/components/SharingPermission/SharingPermissionsTable.tsx

```diff
@@ -53,10 +53,16 @@ function DeletePermissionDialog({
   };
 
   const showDeletionNotice = (participant: SharingSiteWithSource) => {
-    const remainSources = participant.addedBy.filter((source) => source !== MANUALLY_ADDED);
-    if (remainSources.length) {
+    const remainSources = participant.addedBy.filter(
+      (source) => source !== MANUALLY_ADDED
+    ) as ClientType[];
+    const remainSourceDescriptions = remainSources.map((x) => ClientTypeDescriptions[x]);
+    if (remainSourceDescriptions.length) {
       return (
-        <span> (This site will remain shared by {formatStringsWithSeparator(remainSources)})</span>
+        <span>
+          {' '}
```

Comment:
Nit: do we need this space?

Reply:
yeah otherwise it looks like this: 
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/44b082f0-0a6e-49e0-b973-d9f01ab583d9)

Reply:
it used to have a space but now we need to do it this way because the contents of the span no longer fit on one line